whisper-fine-tuning-sprint/README.md

```diff
@@ -0,0 +1,635 @@
+evaluate the speech recognition system on both Common Voice's `"test"` data as well as the *real-world* audio 
+data. Please read through the [Evaluation](#evaluation) section to make sure your evaluation script is in the correct format. Speech recognition systems
+with evaluation scripts in an incorrect format can sadly not be considered for the Challenge.
+
+At the end of the event, the best performing speech recognition system 
+will receive a prize 🏆 - more information regarding the prizes can be found under [Prizes](#prizes).
+
+We believe that framing the event as a competition is more fun, but at the core, the event is about
+creating speech recognition systems in as many languages as possible as a community.
+This can be achieved by working together, helping each other to solve bugs, share important findings, etc...🤗
+
+**Note**:
+Please, read through the section on [Communication & Problems](#communication-and-problems) to make sure you 
+know how to ask for help, etc...
+All important announcements will be made on discord. Please make sure that 
+you've joined [this discord channel](https://hf.co/join/discord)
+
+Also, please make sure that you have been added to the [Speech Event Organization](https://CREATE_AN_ORGANISATION_ON_HUB). 
+You should have received an invite by email. If you didn't receive an invite, please contact the organizers, *e.g.* Sanchit or VB directly on discord.
+
+[comment]: # VB: create an organisation on the hub.
+
+## Important dates
+
+![timeline](https://github.com/patrickvonplaten/scientific_images/raw/master/Robush%20Speech%20Challenge.png)
+
+[comment]: # VB - Create an infographic for the timeline.
+
+## Data and preprocessing
+
+In this section, we will quickly go over how to find suitable training data and 
+how to preprocess it.
+
+To begin with, **all data except Common Voice's `"test"` data can be used as training data.**
+The exception includes all Common Voice versions as the test data split of later Common Voice versions often
+overlaps with the one of previous versions, *e.g.* the test data of Common Voice 7 in English is 
+to a big part identical to the test data of Common Voice 6 in English:
+
+```python
+load_dataset("mozilla-foundation/common_voice_11_0", "en", split="test") 
+```
+
+includes more or less the same data as
+
+```python
+load_dataset("mozilla-foundation/common_voice_10_0", "en", split="test") 
+```
+
+However, we strongly encourage participants to make use of Common Voice's other splits, *e.g.* `"train"` and `"validation"`.
+For most languages, the Common Voice dataset offers already a decent amount of training data. It is usually 
+always advantageous to collect additional data. To do so, the participants are in a first step encouraged to search the
+Hugging Face Hub for additional audio data, for example by selecting the category 
+["speech-processing"](https://huggingface.co/datasets?task_categories=task_categories:speech-processing&sort=downloads).
```

Comment:
https://huggingface.co/datasets?task_categories=task_categories:automatic-speech-recognition is better :) 

```suggestion
["automatic-speech-recognition"](https://huggingface.co/datasets?task_categories=task_categories:automatic-speech-recognition).
```